database_backup/database_backup.task.yaml

```
@@ -0,0 +1,21 @@
+slug: demo_database_backup
+name: Database backup
+resources:
+  db: demo_db
```

Do we usually add comments, e.g.

```suggestion
  # Replace this with the slug of your DB resource.
  db: demo_db
```

---

We've gone back and forth on this, but I think it makes sense here.